### PR TITLE
Improve user lookup form field query

### PR DESCRIPTION
**Pull request by smithellis**
* Remove from loop
* Get all responses, not just first

Attempting to correct Stage issues with https://github.com/mozilla/sumo/issues/2019 - I think the issue is related related to speed. Improving this query might resolve.

### CODE REVIEW

File: kitsune/sumo/form_fields.py [hunk 1]
```diff
@@ -67,15 +67,14 @@ def to_python(self, value):
...
-        for username in value.split(","):
-            username = username.strip()
-            if username:
-                user = User.objects.filter(
-                    Q(username=username) | Q(profile__name=username)
-                ).first()
-                if user:
-                    if user.is_active:
-                        users.append(user)
+        usernames = [u.strip() for u in value.split(",") if u.strip()]
```
Comment (akatsoulas) [inline kitsune/sumo/form_fields.py:70]: Good idea to remove the `for` loop but isn't his already a list of usernames? Additionally why is `if u.strip()` needed?

Comment (smithellis) [reply kitsune/sumo/form_fields.py:70]: Lordy. That last u.strip is not needed.

Comment (smithellis) [reply kitsune/sumo/form_fields.py:70]: It isn't a list of usernames originally - it's whatever the value of the form field is as the user types. So the flow is get that value, convert to a clean list of strings, get a queryset of any User instance that matches on username or profile.name, validate that the user is active then return the list of User objects.

Comment (akatsoulas) [reply kitsune/sumo/form_fields.py:72]: I mean why are you validating the u.strip. This should be 
```python
usernames = [u.strip() for u in values.split() if u]
```

Comment (akatsoulas) [reply kitsune/sumo/form_fields.py:72]: I would also go for something more descriptive that `u`

**Review by akatsoulas — APPROVED** [review]
r+wc

File: kitsune/sumo/form_fields.py (hunk 1 above)
Comment (smithellis) [reply kitsune/sumo/form_fields.py:72]: Oh - got it. Thanks!

**Pull request merged**